### PR TITLE
Fix console container propTypes

### PR DESCRIPTION
**Pull request by pgalic96**
#### Summary

References #1086 

#### Changes

- Added new prop-type `formatters`
- Adjust existing and add new proptypes for `webui/console/containers/*`

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, database and configuration, according to the stability commitments in `README.md`.
- [x] Testing: The changes are covered with unit tests. The changes are tested manually as well.
- [ ] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [x] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

File: pkg/webui/console/containers/api-keys-table/index.js [hunk 1]
```diff
@@ -73,6 +73,17 @@ const headers = [
 ]
 
 export default class ApiKeysTable extends Component {
+  static propTypes = {
+    baseDataSelector: PropTypes.func.isRequired,
+    entityId: PropTypes.string,
```
Comment (kschiffer) [inline pkg/webui/console/containers/api-keys-table/index.js]:
```suggestion
    entityId: PropTypes.string.isRequired,
```

File: pkg/webui/console/containers/events-subscription/events-subscription.js [hunk 2]
```diff
@@ -20,6 +20,24 @@ import PropTypes from '../../../lib/prop-types'
...
+    onClear: PropTypes.func,
+    onRestart: PropTypes.func.isRequired,
+    toAllUrl: PropTypes.string,
+    widget: PropTypes.bool,
+  }
+
+  static defaultProps = {
+    widget: false,
+    onClear: () => null,
+    toAllUrl: null,
```
Comment (kschiffer) [inline pkg/webui/console/containers/events-subscription/events-subscription.js]:
```suggestion
    toAllUrl: undefined,
```

File: pkg/webui/console/containers/fetch-table/index.js [hunk 3]
```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
     this.debounceCancel = cancel
   }
 
+  static propTypes = {
```
Comment (kschiffer) [inline pkg/webui/console/containers/fetch-table/index.js]: Move to top.

File: pkg/webui/console/containers/fetch-table/index.js [hunk 4]
```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
...
+    addMessage: PropTypes.message,
+    fetching: PropTypes.bool,
```
Comment (kschiffer) [inline pkg/webui/console/containers/fetch-table/index.js]: `baseDataSelector` should also be here.

File: pkg/webui/console/containers/fetch-table/index.js [hunk 5]
```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
...
+        name: PropTypes.string.isRequired,
+        icon: PropTypes.string,
+        disabled: PropTypes.bool,
+      }),
+    ),
+    totalCount: PropTypes.number,
+  }
+
+  static defaultProps = {
+    pageSize: 20,
```
Comment (kschiffer) [inline pkg/webui/console/containers/fetch-table/index.js]: Refer to `lib/page-sizes.js` here.

File: pkg/webui/console/containers/fetch-table/index.js [hunk 6]
```diff
@@ -89,6 +90,58 @@ class FetchTable extends Component {
...
+        centered: PropTypes.bool,
+        sortable: PropTypes.bool,
+        width: PropTypes.number,
+      }),
+    ),
+    itemPathPrefix: PropTypes.string,
+    items: PropTypes.arrayOf(PropTypes.shape({})),
+    mayAdd: PropTypes.bool,
+    pageSize: PropTypes.number,
+    pathname: PropTypes.string,
```
Comment (kschiffer) [inline pkg/webui/console/containers/fetch-table/index.js]:
```suggestion
    pathname: PropTypes.string.isRequired,
```
Should work, I believe.

File: pkg/webui/console/containers/webhooks-table/index.js [hunk 7]
```diff
@@ -55,6 +56,10 @@ export default class WebhooksTable extends React.Component {
     this.getWebhooksList = () => getWebhooksList(appId)
   }
 
+  static PropTypes = {
```
Comment (kschiffer) [inline pkg/webui/console/containers/webhooks-table/index.js]: Move to top.

File: pkg/webui/console/containers/pubsubs-table/index.js [hunk 8]
```diff
@@ -83,6 +84,10 @@ export default class PubsubsTable extends React.Component {
     this.getPubsubsList = () => getPubsubsList(appId)
   }
 
+  static propTypes = {
```
Comment (kschiffer) [inline pkg/webui/console/containers/pubsubs-table/index.js]: Move to top.

File: pkg/webui/console/containers/api-keys-table/index.js (hunk 1 above)
Comment (pgalic96) [reply pkg/webui/console/containers/api-keys-table/index.js]: results in failed PropType, `entityId` is undefined

File: pkg/webui/console/containers/fetch-table/index.js (hunk 4 above)
Comment (pgalic96) [reply pkg/webui/console/containers/fetch-table/index.js]: eslint doesn't recognize baseDataSelector as a propType, which means it won't check for it so there is no purpose in defining it I believe. I added it but not sure if its necessary.

File: pkg/webui/console/containers/api-keys-table/index.js (hunk 1 above)
Comment (kschiffer) [reply pkg/webui/console/containers/api-keys-table/index.js]: As far as I can see the `id` prop is not even used by `FetchTable`. So you can delete it alltogether.

File: pkg/webui/console/containers/fetch-table/index.js (hunk 4 above)
Comment (kschiffer) [reply pkg/webui/console/containers/fetch-table/index.js]: Weird? I can set it with no warning
```js
…
baseDataSelector: PropTypes.shape({
items: base[props.entity] || [],
    totalCount: PropTypes.number,
    fetching: PropTypes.bool,
    fetchingSearch: PropTypes.func,
    pathname: PropTypes.string,
    mayAdd: PropTypes.bool,
}).isRequired,
…

**Pull request merged**